Osiris/Hacks/Animations.cpp

```diff
@@ -71,7 +71,7 @@ void Animations::fake() noexcept
         localPlayer.get()->InvalidateBoneCache();
         memory->setAbsAngle(localPlayer.get(), Vector{ 0, fakeanimstate->GoalFeetYaw, 0 });
         std::memcpy(localPlayer.get()->animOverlays(), &networked_layers, sizeof(AnimationLayer) * localPlayer->getAnimationLayerCount());
-
+        localPlayer.get()->getAnimationLayer(12)->weight = std::numeric_limits<float>::epsilon();
```

Comment:
Lmao is this a meme?

Reply:
uhh
https://www.unknowncheats.me/forum/counterstrike-global-offensive/386219-animation-related-information.html